### PR TITLE
Fixes issue with stripping '.' directory

### PR DESCRIPTION
**Pull request by ForestEckhardt**
Strip components would string `.` off of the path when stripping components. This leads to some hard to parse behavior by the end user as it appears that strip components does nothing. The path is now cleaned before stripping to ensure more expected behavior.

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [x] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by sophiewigmore** [comment]
@ForestEckhardt Is there a test you could add for this?

**Review by ryanmoran — APPROVED** [review]

**Comment by ForestEckhardt** [comment]
@sophiewigmore Test has been added ready for your approval

**Comment by ForestEckhardt** [comment]
@sophiewigmore bump

**Review by sophiewigmore — APPROVED** [review]
